spanner/api/Spanner.Samples/UpdateDataWithJsonbAsyncPostgre.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_postgresql_jsonb_update_data]
+
+using Google.Cloud.Spanner.Data;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+public class UpdateDataWithJsonbAsyncPostgreSample
+{
+    public async Task UpdateDataWithJsonbAsyncPostgre(string projectId, string instanceId, string databaseId)
+    {
+        List<VenueInformation> venueInformationList = new List<VenueInformation>
+        {
+            // If you are using .NET Core 3.1 or later, you can use System.Text.Json for serialization instead.
+            new VenueInformation
+            {
+                VenueId = 19,
+                Details = JsonConvert.SerializeObject(new
+                {
+                    rating = 9,
+                    open = true,
+                })
+            },
+            new VenueInformation
+            {
+                VenueId = 4,
+                // PostgreSQL JSONB sorts first by key length and then lexicographically with equivalent key length 
```

Comment:
I don't understand what this comment is about.
Does key here mean field?
Also, I don't think what's described in this comment is represented on the example below, which makes the comment a little out of place.

Reply:
Sure Amanda. Does this sound better? "PostgreSQL JSONB takes the last value in the case of duplicate fields".

Reply:
We have discussed this offline and agreed on a couple changes around this comment.